source/settings/core.rst

```diff
@@ -68,11 +68,17 @@ See :ref:`settings` for list of all setting groups.
 
 
 .. dovecot_core:setting:: auth_debug
+   :changed: v2.4.0;v3.0.0
    :default: no
    :values: @boolean
 
-   Enables all authentication debug logging (also enables
-   :dovecot_core:ref:`auth_verbose`). Passwords are logged as ``<hidden>``.
+   Enables all authentication debug logging.
+   Passwords are logged as ``<hidden>``.
```

Comment:
You can leave the "(also enables auth_verbose)" since it seems to be true still.